docs/readme/body.rst

```diff
@@ -554,6 +557,46 @@ Environment Variables
     ``AWSCLI_LOGIN_ROOT`` is set to ``/tmp`` then the plugin will
     look for configuration files in (``/tmp/.aws-login/``).
 
+Keyrings with WSL and Windows Credential Store
+=====================
+
+If running under WSL, you may wish to store credentials in the Windows
+Credential Store in the Windoes host operating system and access them
+fromwithin WSL. To do this, first install Python on the Windows
+host operating system if not already installeed. You can do this from
+the Microsoft Store or using Winget, or you can download the installer
+directly from the Python official site. The Python keyring module
+will need to beinstalled on the Windows Python instance. From a
+Windows command prompt, run:
+
+    pip install keyring
+
+Next, inside WSL, install the keyring-pybridge module:
+
+    pip install keyring-pybridge
+
+Still in WSL, Set the PYTHON_KEYRING_BACKEND environment variable to
+tell Python to use the pybridge keyring backend. This is shell
+specific but might look like:
+
+    export PYTHON_KEYRING_BACKEND=keyring_pybridge.PyBridgeKeyring
+
+Finally, set the KEYRING_PROPERTY_PYTHON envorinment variableto point
+to the Windows Python executable. If you need to find the full path
+to this executable, run this from the Windows command prompt:
+
+    where python.exe
```

Comment:
```suggestion
    which python.exe
```

When I was stepping through, I had to swap this for `which`.

Reply:
> When I was stepping through, I had to swap this for `which`.

That's because you ran it inside WSL. The instructions say to run that from a Windows command prompt. Which is not a Windows command, but where is. If you do have which on Windows, I'd like to know where you got it.

Reply:
@kwessel Oh! Since my default shell is `PowerShell`, `where` did not work for me. It works fine in Windows Command Prompt. In `PowerShell`, `Get-Command python.exe` works - but still requires translation.

A nice thing about running `which python.exe` inside WSL, is that it gave me the WSL-friendly path, without any translation. It only works because I included the `.exe`, since `which python` stops at the version inside WSL.

Reply:
which python.exe doesn't work for me inside wsl. It can't find python.exe, presumably because I don't have any /mnt/c directories in my search path. I imagine most people won't. I went with the where command since that was a lowest common denominator. One could also do /find /mnt/c -name python.exe inside wsl, but that takes a very long time to run and produces a lot of noise. So, I went with the Windows command prompt and the "where" command.

Reply:
That's a great point. I did have all of `/mnt` on my path, but I do typically remove it. 

@ddriddle `which python.exe` worked for me because I had not removed the weird `/mnt` path stuff from my profile yet. 

The approach @kwessel included will be more reliable. I made a separate suggestion for emphasis to help others not miss that the command is for Windows Command Prompt.